node_modules/nodeos-barebones/cpio.txt

```diff
@@ -27,9 +27,9 @@ slink /lib/ld-musl-$CPU_FAMILY.so.1 libc.so                    0555 0 0
 # gcc
 
 file /lib/libgcc_s.so.1       $TOOLS/$TARGET/lib/libgcc_s.so.1       0444 0 0
-file /lib/libstdc++.so.6.0.17 $TOOLS/$TARGET/lib/libstdc++.so.6.0.17 0444 0 0
+file /lib/libstdc++.so.6.0.21 $TOOLS/$TARGET/lib/libstdc++.so.6.0.21 0444 0 0
 
-slink /lib/libstdc++.so.6 libstdc++.so.6.0.17 0444 0 0
+slink /lib/libstdc++.so.6 libstdc++.so.6.0.21 0444 0 0
```

Comment:
Thanks for remind me this, we need to find a better solution for versioned libraries :-) Maybe they could be injected the same way it's done with the `$TOOLS` and `$TARGET` variables...